### ISSUE TITLE
Questions Regarding PasteBar App Features

### ISSUE THREAD

**Issue by mycwcgr** [PasteBar/PasteBarApp#196]
PasteBar is a great app, but I have a couple of questions:

1. Is it possible to export and import categories and items? I would like to share the information with others.

2. Can I disable the automatic pasting of items? I would prefer to paste the item manually, for example, by using Ctrl+V.

Thank you for your help!

**Comment by kurdin** [comment]
@mycwcgr Thank you for using PasteBar! 

1. Currently no, but I'll work on the export/import feature when I have time. I have other users that requested this feature, and I think it will be useful. 

2. Are you talking about the Quick Paste window ? In the main PasteBar window, you can double-click on the history item or the saved clip to copy its content to the clipboard, and you can use CTRL+V to paste it anywhere you want. In the Quick Paste window, I have only the click-to-paste feature for now, but I have another user request to implement the copy-to-clipboard option instead of paste with an extra key press. I'll work on it as well in the near future. 

Thanks again for your comments.

**Comment by mycwcgr** [comment]
Thanks for your answer.

When I click the PasteBar icon on the Windows toolbar and select an item from the popup menu, the item's content is copied to the clipboard. Then, when I open Notepad, the content is automatically pasted after about a 1-second delay. However, sometimes it takes a long time to paste, and I have to manually use Ctrl+V. Occasionally, double content is pasted into Notepad.

I would like an option to disable the automatic paste function so that I can always use Ctrl+V for manual pasting instead.

![Ask_Again](https://github.com/user-attachments/assets/eb37cc92-aba1-4dbf-8527-4a3044e2d63a)

**Comment by kurdin** [comment]
@mycwcgr 
Yeah, I understand your use case. I'll think about how to add an option to disable auto pasting from the tray menu. However, an easy workaround for now would be to simply click the item in the tray menu and wait for 1 second or longer, then open Notepad or any other place you want to paste and press Ctrl + V for manually pasting. Also, you can increase the time to paste from 1 second to 2, 3, 4, or 5 seconds as well. I think I just need to add another option there as `disable paste` and make it selectable to disable this feature. 

<img width="500" alt="image" src="https://github.com/user-attachments/assets/aef0152d-b8b4-4985-800c-8bfffa1e9218" />

**Comment by mycwcgr** [comment]
Thanks for your response.

I believe the issue is caused by the current implementation of the automatic paste function.

For a skilled user, a 1-second delay is too long. As a result, I often press Ctrl+V to paste the selected item manually before the 1-second delay is over. However, sometimes PasteBar pastes the content again automatically, resulting in duplicate content.

I think there are two possible solutions to this issue:

1. Add a "0-second" delay option, allowing users to always paste manually without any automatic delay.
2. Add an option to disable the automatic paste function entirely, so users can rely solely on Ctrl+V for pasting.



Additionally, the current setting for "Paste Delay ... 1s" can be misleading.  The app will set the automatic paste function disabled after 1 second，but I think the app will paste an item after 1s automatically and never disable  automatically paste function.


"However, an easy workaround for now would be to simply click the item in the tray menu and wait for 1 second or longer, then open Notepad or any other place you want to paste and press Ctrl + V for manually pasting. "

**Comment by dmegatool** [comment]
Would love to be able to disable the auto pasting too ! +1 :)

**Comment by kurdin** [comment]
implemented #247
